metrics/github/github.py

```diff
@@ -108,6 +108,34 @@ def from_dict(cls, data, repo):
         )
 
 
+@dataclass(frozen=True)
+class Codespace:
+    org: str
+    # The Repo type requires fields neither returned by the codespaces
+    # endpoint, nor required for codespaces metrics so str for repo name
```

Comment:
You don't need to document this, now that `repo` is consistently an instance of `Repo` and `repo_name` is consistently an instance of `str`.